reference/kube-controllers/configuration.md

```diff
@@ -18,7 +18,7 @@ The `{{page.imageNames["calico/kube-controllers"]}}` container includes the foll
 1. namespace controller: watches namespaces and programs {{site.prodname}} profiles.
 1. serviceaccount controller: watches service accounts and programs {{site.prodname}} profiles.
 1. workloadendpoint controller: watches for changes to pod labels and updates {{site.prodname}} workload endpoints.
-1. node controller: watches for the removal of Kubernetes nodes and removes corresponding data from {{site.prodname}}.
+1. node controller: watches for the removal of Kubernetes nodes and removes corresponding data from {{site.prodname}}, and watches for node updates to optionally create and sync host endpoints for each node.
```

Comment:
Not sure, but I believe this is what we mean...

node controller: watches for the removal of Kubernetes nodes and removes corresponding data from {{site.prodname}}, and optionally watches for node updates to create and sync host endpoints for each node.

Reply:
Thanks, added `optionally`. Technically, the feature still runs even when disabled (so that it can cleanup after itself) but from a user's perspective yeah